torchci/lib/bot/triggerInductorTestsBot.ts

```diff
@@ -0,0 +1,66 @@
+import { Context, Probot } from "probot";
+
+export default function triggerInductorTestsBot(app: Probot): void {
+  const preapprovedUsers = ["pytorchbot", "PaliC"]; // List of preapproved users
```

Comment:
Long-term, I think we want to have a https://github.com/pytorch/pytorch/blob/main/.github/pytorch-probot.yml on `triton-lang/triton` to configure the bot from there like the list of pre-approved users here

Reply:
Yeah I think this is better in the long run as well!